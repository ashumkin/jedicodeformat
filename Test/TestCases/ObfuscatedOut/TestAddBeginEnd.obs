unit testaddbeginend;interface implementation procedure testif;var str1:string;bfoo
:boolean;ifish:integer;begin if bfoo then str1:='';if bfoo then begin str1:='';end
;if bfoo then str1:='troo'else str1:='fals';if bfoo then begin str1:='troo';end else
str1:='fals';if bfoo then begin str1:='troo';end else begin str1:='fals';end;if bfoo
then str1:='troo'else begin str1:='fals';end;end;procedure testcases;var str1:string
;bfoo:boolean;ifish:integer;begin case ifish of 1:bfoo:=true;2:begin bfoo:=false;
end;else bfoo:=false;end;case ifish of 1:bfoo:=true;else begin bfoo:=false;end end
;case ifish of 1:bfoo:=true;else bfoo:=false;str1:='troo';end;end;procedure testloop
;var lbtest:boolean;sfoo:string;liloop:integer;begin while lbtest do lbtest:=not 
lbtest;for liloop:=0 to 10 do sfoo:=sfoo+'-';end;end.