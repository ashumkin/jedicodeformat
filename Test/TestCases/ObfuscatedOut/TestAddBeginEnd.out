unit TestAddBeginEnd;

interface

implementation

procedure TestIf;
var
  Str1:  string;
  bFoo:  boolean;
  iFish: integer;
begin
  if bFoo then
    Str1 := '';

  if bFoo then
  begin
    Str1 := '';
  end;

  if bFoo then
    Str1 := 'troo'
  else
    Str1 := 'fals';

  if bFoo then
  begin
    Str1 := 'troo';
  end
  else
    Str1 := 'fals';

  if bFoo then
  begin
    Str1 := 'troo';
  end
  else
  begin
    Str1 := 'fals';
  end;

  if bFoo then
    Str1 := 'troo'
  else
  begin
    Str1 := 'fals';
  end;
end;

procedure TestCases;
var
  Str1:  string;
  bFoo:  boolean;
  iFish: integer;
begin

  case iFish of
    1:
      bFoo := true;
    2:
    begin
      bFoo := false;
    end;
    else
      bFoo := false;
  end;


  case iFish of
    1:
      bFoo := true;
    else
    begin
      bFoo := false;
    end
  end;

  case iFish of
    1:
      bFoo := true;
    else
      bFoo := false;
      Str1 := 'troo';
  end;

end;

procedure TestLoop;
var
  lbTest: boolean;
  sFoo:   string;
  liLoop: integer;
begin
  while lbTest do
    lbTest := not lbTest;

  for liLoop := 0 to 10 do
    sFoo := sFoo + '-';
end;

end.
 